h/groups/util.py

```diff
@@ -33,3 +33,7 @@ def pubid(self):
     @property
     def is_public(self):
         return True
+
+    @property
+    def creator_id(self):
+        return None
```

Comment:
Putting `__world__` into the database is something that we should probably test quite a bit more than just as a small follow-up PR. I'd rather have it as a proper card in a sprint than something we just do on the side and then it'll turn into a moderately sized chunk of work.